scrapers/FFstockings.yml

```diff
@@ -0,0 +1,24 @@
+name: FFstockings
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - ffstockings.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="trailerContent"]/h2
+      Performers:
+        Name: //li/a/span
+      Tags:
+        Name:
+          selector: //div[@class="tags"]/a
+      Details:
+        selector: //div[@class="trailerContent"]/p
```

Comment:
it seems that in scenes with free trailers (eg `http://ffstockings.com/tour2/trailers/Cum-On-My-Face.html`) the trailer info gets in the description
keeping only the plain  text part of the p seems to work (with very limited testing)
```yaml
        selector: //div[@class="trailerContent"]/p/text()
```